config/runtime.exs

```diff
@@ -31,6 +31,38 @@ case System.get_env("MAPBOX_PRIMARY_TOKEN") do
       mapbox_public_token: System.get_env("MAPBOX_PUBLIC_TOKEN")
 end
 
+database_name_raw = System.get_env("DATABASE_NAME")
+
+database_name =
+  if config_env() == :test do
+    # Convenience so that we can switch between running application
+    # And running tests without having to change environment variables
+    database_name_raw <> "_test"
+  else
+    database_name_raw
+  end
```

Comment:
This is a little convenience I remember appreciating [from draft ](https://github.com/mbta/draft/blob/e43dd5bc66de4f7a9f212ada0f1a78a4a87f1de7/config/runtime.exs#L5) to avoid accidentally running tests against the dev DB with conflicting data.